src/test/java/com/wikia/webdriver/common/core/helpers/Emulator.java

```diff
@@ -1,11 +1,50 @@
 package com.wikia.webdriver.common.core.helpers;
 
-public class Emulator {
+import java.util.Map;
 
-  public static final String GOOGLE_NEXUS_5 = "Google Nexus 5";
+import lombok.Getter;
 
-  public static final String APPLE_IPHONE_6 = "Apple iPhone 6";
+import com.google.common.collect.ImmutableMap;
 
-  private Emulator() {
+public enum Emulator {
+  GOOGLE_NEXUS_5("Google Nexus 5"),
+  APPLE_IPHONE_6("Apple iPhone 6"),
+  DESKTOP_BREAKPOINT_BIG(
+      new ImmutableMap.Builder<String, Object>()
+          .put("width", 1296)
+          .put("height", 900)
+          .put("pixelRatio", 1.0)
+          .put("touch", false)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "touch" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)